content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

```diff
@@ -521,7 +521,214 @@ To verify your device status, click on your FoundriesFactory, go to **Devices**
+
+If you are to use a carrier, please check the carrier's configuration to be paired with Portenta X8. For **Portenta Max Carrier**, `BOOT SEL` and `BOOT` must be set to ON position with DIP switches as shown in the figure:
+
+![Portenta Max Carrier DIP switches](assets/max-carrier-dip-switches.png)
+
+For **Portenta Breakout**, `BT_SEL` and `BOOT`DIP switches must be set to ON position as well as shown in the figure:
+
+![Portenta Breakout DIP switches](assets/breakout-dip-switches.png)
+
+If you decide to flash Portenta X8 without using the carrier, use the following command sequence inside the Portenta X8's terminal via ADB while you are in the root environment with root permission to reset Portenta X8's bootloader sector:
+
+```arduino
+echo 0 > /sys/block/mmcblk2boot0/force_ro
+```
+
+```arduino
+dd if=/dev/zero of=/dev/mmcblk2boot0 bs=1024 count=4096 && sync
+```
+
+```arduino
+echo 0 > /sys/block/mmcblk2boot1/force_ro
+```
+
+```arduino
+dd if=/dev/zero of=/dev/mmcblk2boot1 bs=1024 count=4096 && sync
+```
+
+Now that we have the Portenta X8 in programming mode, we need to flash the OS image. Within the previously described OS image file structure, you need to navigate to `mfgtool-files-portenta-x8` directory. Inside the directory, you will find the `uuu` executable and its components. Here, you will open a terminal and run the following command:
+
+```
+uuu full_image.uuu
+```
+
+It requires one additional step to flash Portenta X8 without a carrier. You will execute the command **first** to let it search for the board. Meanwhile, you will recycle the power source for Portenta X8 by unplugging and reconnecting the USB-C® cable. It will let the board begin its boot sequence, allowing it to enter programming mode as set with the defaulted internal bootloader. When the active `uuu` instance detects board is in programming mode, it will continue with its flashing process.
+
+Once the flashing operation finishes, you will be greeted with a similar message in the terminal as the following figure:
+
+![Successful uuu flashing operation](assets/uuu-flashing-success.png)
+
+This applies to both flashing scenarios. If you have the carrier attached and decide to continue using docked with the platform, you will have to reset the DIP switch positions for either `BOOT SEL` or `BT_SEL` and `BOOT` to OFF state. Reconnect the board and wait approximately 10 seconds until Blue LED starts blinking, confirming the boot was successful.
+
+In case the Portenta X8 was flashed barebone, you will just need to recycle the power and should be ready with the latest OS image.
+
+***For more in-depth tutorial for flashing Portenta X8, please check out [How To Flash Your Portenta X8](https://docs.arduino.cc/tutorials/portenta-x8/image-flashing) tutorial.***
+
+### Portenta X8 Post-Flash Operation
```

Comment:
Why are we added this section here? I find it quite confusing and it's already described in the out of the box section + Working with Portenta X8 Manager section

Reply:
I have added this section as a way to register the board using the command line. The idea was similar to the "How to Flash Your Portenta X8" tutorial for post-flash, but since we could confuse the users as you mentioned in the other [PR here](https://github.com/arduino/docs-content/pull/948#discussion_r1149592357), we could remove the section without issue. 🙂